*** unilbrk.in.h.orig	2008-05-10 14:46:17.000000000 +0200
--- unilbrk.in.h	2009-04-10 22:39:40.000000000 +0200
***************
*** 24,30 ****
  #include "unitypes.h"
  
  /* Get locale_charset() declaration.  */
! #include "localcharset.h"
  
  
  #ifdef __cplusplus
--- 24,30 ----
  #include "unitypes.h"
  
  /* Get locale_charset() declaration.  */
! #include <unistring/localcharset.h>
  
  
  #ifdef __cplusplus
