*** lib/unistr.in.h.orig	2021-08-01 16:43:54.915090697 +0200
--- lib/unistr.in.h	2021-08-01 15:52:07.388856862 +0200
***************
*** 19,26 ****
  
  #include "unitypes.h"
  
  /* Get bool.  */
! #include <stdbool.h>
  
  /* Get size_t, ptrdiff_t.  */
  #include <stddef.h>
--- 19,32 ----
  
  #include "unitypes.h"
  
+ /* Get common macros for C.  */
+ #include <unistring/cdefs.h>
+ 
+ /* Get inline if available.  */
+ #include <unistring/inline.h>
+ 
  /* Get bool.  */
! #include <unistring/stdbool.h>
  
  /* Get size_t, ptrdiff_t.  */
  #include <stddef.h>
